client/components/pie-chart/style.scss

```diff
@@ -0,0 +1,80 @@
+.pie-chart__chart-section-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__chart-section-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__chart-section-2 {
+	fill: $blue-light;
+}
+
+.pie-chart__legend-sample-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__legend-sample-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__legend-sample-2 {
+	fill: $blue-light;
+}
+
+.pie-chart__chart {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.pie-chart__legend-item {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+}
```

Comment:
Disabling `flex` here doesn't seem to have any effect, which leads me to believe we could get rid of it.